reference_simulations/Description/reference_simulations_1_summary.tex

```diff
@@ -12,7 +12,7 @@
 
 \title{pyuvsim Reference Simulations \\ version 1}
 
```

Comment:
This file will be removed with PR #262 , since this description tex file is redundant with the markdown file.